source/game.roomhandler.news.bmx

```diff
@@ -930,6 +931,21 @@ Type RoomHandler_News extends TRoomHandler
 	End Function
 
 
+	Function onTryDropNews:int(triggerEvent:TEventBase)
+		local guiNews:TGUINews = TGUINews( triggerEvent._sender )
+		local receiverList:TGUIListBase = TGUIListBase( triggerEvent._receiver )
+		if not guiNews or not receiverList then return FALSE
+
+		if not GetPlayerBaseCollection().IsPlayer(guiNews.news.owner) return False
+		'prevent using news that cannot be afforded
+		if receiverList = guiNewsListUsed
+			if Not guiNews.news.paid and Not GetPlayerBase().getFinance().canAfford(guiNews.news.GetPrice( GetPlayerBase().playerID ))
```

Comment:
Der Owner-Check sollte denke ich auch sicherstellen, dass man nicht mit dem Generalschluessel die News anderer Spieler platzieren kann.

Reply:
Durch den "Force Push" ist die vorherige Variante nicht mehr sichtbar.

Reply:
D.h. ich ergänze noch ein Veto wenn der Spieler und der News-Owner unterschiedlich sind.

Reply:
Genau

Reply:
Gerade getestet. Mit Generalschlüssel in einem anderen Studio lassen sich die Nachrichten nicht aufheben. Insofern wäre das nur ein *zusätzliches* Sicherheitsnetz.
Außerdem scheint der aktuelle Spieler nicht im Event zur Verfügung zu stehen. Weder playerID noch channel scheinen gesetzt zu sein...